packages/analytics-core/test/plugins/destination.test.ts

```diff
@@ -560,16 +599,48 @@ describe('destination', () => {
         reset: async () => undefined,
         getRaw: async () => undefined,
       };
+      const event1 = { event_type: 'event', insert_id: '1' };
+      const event2 = { event_type: 'filtered_event', insert_id: '2' };
+      const events = [event1, event2];
+      const eventsToAdd = events.map((event) => {
+        return {
+          event,
+          attempts: 0,
+          callback: () => undefined,
+          timeout: 0,
+        };
+      });
       const set = jest.spyOn(destination.config.storageProvider, 'set').mockResolvedValueOnce(undefined);
-      destination.saveEvents();
+      destination.queue = eventsToAdd;
+      const eventsToRemove = [eventsToAdd[1]];
+      destination.removeEvents(eventsToRemove);
       expect(set).toHaveBeenCalledTimes(1);
+      expect(set).toHaveBeenCalledWith('', expect.objectContaining([event1]));
     });
 
-    test('should be ok with no storage provider', () => {
```

Comment:
Can we keep the test as we keep the code where it tests on?

Reply:
It has been moved into line 584, what do you mean?

Reply:
I didn't see it was moved, I though this test was deleted somehow